p2p/peerManager/peerManager.go

```diff
@@ -122,19 +127,30 @@ func NewManager(ctx context.Context, low int, high int, datastore datastore.Data
 		return nil, err
 	}
 
-	bestPeersDB, err := peerdb.NewPeerDB(c_bestDBName)
-	if err != nil {
-		return nil, err
-	}
-
-	responsivePeersDB, err := peerdb.NewPeerDB(c_responsiveDBName)
-	if err != nil {
-		return nil, err
-	}
-
-	lastResortPeers, err := peerdb.NewPeerDB(c_lastResortDBName)
-	if err != nil {
-		return nil, err
+	// Initialize the expected peerDBs
+	peerDBs := make(map[string][]*peerdb.PeerDB)
+	for _, loc := range utils.GetRunningZones() {
+		domLocations := loc.GetDoms()
+		for _, domLoc := range domLocations {
+			domLocName := domLoc.Name()
+			if peerDBs[domLocName] != nil {
+				// This peerDB has already been initialized
+				continue
+			}
+			peerDBs[domLocName] = make([]*peerdb.PeerDB, 3)
```

Comment:
what is this 3 ?

Reply:
3 is for the DBs (best, responsive, and all)